features/step_definitions/resource_steps.rb

```diff
@@ -175,9 +182,22 @@
   end
 end
 
-Given /^the current account has (\d+) (?:([\w+]+) )?"([^\"]*)" (?:for|in) (?:all|each) "([^\"]*)"$/ do |count, traits, resource, association|
-  associated_records = @account.send(association.pluralize.underscore)
-  traits             = traits&.split()&.map(&:to_sym)
+Given /^the current account has (\d+) (?:([\w+]+) )?"([^\"]*)" (?:with|for|in) (?:all|each) "([^\"]*)"$/ do |count, traits, resource, association|
+  associated_records =
+      case association.underscore.pluralize
+      when 'processes'
+        @account.machine_processes
+      when 'artifacts'
+        @account.release_artifacts
+      when 'packages'
+        @account.release_packages
+      when 'engines'
+        @account.release_engines
```

Comment:
If these associations were named correctly on the account, this wouldn't be needed.